tests/integration/ha_tests/helpers.py

```diff
@@ -46,33 +52,33 @@ class ProcessRunningError(Exception):
     """Raised when a process is running when it is not expected to be."""
 
 
-def replica_set_client(replica_ips: List[str], password: str, app=APP_NAME) -> MongoClient:
+def replica_set_client(replica_ips: List[str], password: str, app_name=None) -> MongoClient:
```

Comment:
I believe the `app_name` cannot be None

```suggestion
def replica_set_client(replica_ips: List[str], password: str, app_name: str) -> MongoClient:
```

Reply:
agreed, what do you think @dmitry-ratushnyy ?

Reply:
Yes, "None" was added as tmp solution, we can remove it. Thank you 😄